README.md

```diff
@@ -1,36 +1,25 @@
-# Check Digit < Insert name > Library
+# Memoize
 
-The Check Digit < Insert Name > library is a library for Check Digit services to deal with [ short description about what the library does]
+Copyright (c) 2021-2023 [Check Digit, LLC](https://checkdigit.com)
 
-Features:
+Memoize is a light-weight utility library used by Check Digit services for memoization purposes.
```

Comment:
I think we should add an explantation that its designed to memoize promises, and evict from the cache anything that rejects.